accepted/PSR-16-simple-cache.md

```diff
@@ -25,8 +25,8 @@ own caching implementations and easily rely on the one given to them by the
 framework, or another dedicated cache library.
 
 PSR-6 solves this problem already, but in a rather formal and verbose way for
-what the most simple use cases need. This simpler approach aims to build a
-standardized streamlined interface for common cases.  It is independent of
+what the most simple use cases need. This simpler approach aims at building a
+standardized streamlined interface for common cases. It is independent of
```

Comment:
"aims to build" is correct grammar in English.

Reply:
I actually saw more of the other one. Both are correct and yet I might be influenced by sports and video games, for "aim at" seems to be used for physical targeting slightly more than the other way of writing it. Might remove that from the PR.